README.md

```diff
@@ -13,7 +13,7 @@ We have nightly and versioned Docker images, Debian packages, RPM packages, and
 
 ## Building From Source
 
-This project requires Go 1.13 and Go module support.
+This project requires Go 1.15 and Go module support.
```

Comment:
A few more places to double-check:
1. `README.md` on `master`
2. `CONTRIBUTING.md` on `master`
3. `CONTRIBUTING.md` on `2.0`

Reply:
(Only `CONTRIBUTING.md` on `2.0` would affect this PR, the others would need to get fixed separately)

Reply:
Good call. I updated a few additional references in the `2.0` branch from `go1.13` to `go1.15` as well - in `CONTRIBUTING.md` and `DEVELOPMENT.md`.

The `master` branch already references `go1.15` in its `README.md` and `CONTRIBUTING.md` so no updates should be needed there - and there isn't a `DEVELOPMENT.md` on master.